showcase/package.json

```diff
@@ -120,8 +120,8 @@
     "@angular/platform-browser-dynamic": "6.0.4",
     "@angular/platform-server": "6.0.4",
     "@angular/router": "6.0.4",
-    "@nationalbankbelgium/stark-core": "file:../dist/packages-dist/stark-core/nationalbankbelgium-stark-core-10.0.0-alpha.2-2c723cb.tgz",
-    "@nationalbankbelgium/stark-ui": "../packages/stark-ui",
+    "@nationalbankbelgium/stark-core": "file:../dist/packages-dist/stark-core/nationalbankbelgium-stark-core-10.0.0-alpha.3-2b732be.tgz",
+    "@nationalbankbelgium/stark-ui": "file:../dist/packages-dist/stark-ui/nationalbankbelgium-stark-ui-10.0.0-alpha.3-2b732be.tgz",
```

Comment:
This line should not be changed otherwise it will brake the showcase assets.

I know that is automatically changed when you build the packages but we will discuss and fix that soon.